core/src/test/java/org/neo4j/ogm/metadata/schema/DomainInfoSchemaBuilderTest.java

```diff
@@ -134,6 +135,12 @@ public void givenSubtypeOfGenericType_thenPamCorrectly() throws Exception {
         assertThat(organisations.type()).isEqualTo("FOUNDED");
         assertThat(organisations.direction(person)).isEqualTo(OUTGOING);
         assertThat(organisations.other(person)).isEqualTo(schema.findNode("Organisation"));
+
```

Comment:
Move to separate test case? This one was for the generic version of `Founded extends Associated<Person, Organisation>`